File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/direct/Java/C/s010698290_comments_removed.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

#define MAX_N 100005

int n, m;
int penal[MAX_N];
int solved[MAX_N];

int main() {
    int i, num;
    char str[10];
    int correct_ans = 0, num_penal = 0;

    scanf("%d %d", &n, &m);

    for (i = 0; i <= n; i++) {
        solved[i] = 0;
        penal[i] = 0;
    }

    for (i = 0; i < m; i++) {
        scanf("%d %s", &num, str);
        if (str[0] == 'A') {
            solved[num] = 1;
        } else {
            if (solved[num] == 0) {
                penal[num]++;
            }
        }
    }

    for (i = 1; i <= n; i++) {
        if (solved[i] == 1) {
            correct_ans++;
            num_penal += penal[i];
        }
    }

    printf("%d %d\n", correct_ans, num_penal);

    return 0;
}
// End of Code

File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/cot/Java/Go/s010698290_comments_removed.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	scanner := bufio.NewScanner(os.Stdin)
	scanner.Split(bufio.ScanWords)

	n, _ := strconv.Atoi(scanner.Text())
	scanner.Scan()
	m, _ := strconv.Atoi(scanner.Text())
	scanner.Scan()

	penal := make([]int, n+1)
	solved := make([]bool, n+1)

	for i := 0; i < m; i++ {
		num, _ := strconv.Atoi(scanner.Text())
		scanner.Scan()
		str := scanner.Text()
		scanner.Scan()
		if str[0] == 'A' {
			solved[num] = true
		} else {
			if !solved[num] {
				penal[num]++
			}
		}
	}

	correct_ans := 0
	num_penal := 0
	for i := 1; i <= n; i++ {
		if solved[i] {
			correct_ans++
			num_penal += penal[i]
		}
	}

	fmt.Println(correct_ans, num_penal)
}

// End of Code
